pages/packages/index.tsx

```diff
@@ -1,11 +1,93 @@
-import DocTemplate from '../docs/[...slug]'
-import { getDocProps } from '../../utils/docs/getDocProps'
+import { DocsGrid, DocGridContent } from '../docs/[...slug]'
 import { GetStaticProps } from 'next'
 import { GithubError } from 'next-tinacms-github'
+import React, { useState } from 'react'
+import { createTocListener } from 'utils'
+import { NextSeo } from 'next-seo'
+import { openGraphImage } from 'utils/open-graph-image'
+import { DocsLayout, MarkdownContent } from 'components/layout'
+import { getJsonPreviewProps, readJsonFile } from 'utils/getJsonPreviewProps'
+import path from 'path'
+
+
+export default function PackageIndex(props) {
+  const excerpt = 'Packages for TinaCMS.'
+
+  const name = "Packages"
+
+  const contentRef = React.useRef<HTMLDivElement>(null)
+  const isBrowser = typeof window !== `undefined`
+  const [activeIds, setActiveIds] = useState([])
+
+  React.useEffect(() => {
+    if (!isBrowser || !contentRef.current) {
+      return
+    }
+    const activeTocListener = createTocListener(contentRef, setActiveIds)
+    window.addEventListener('scroll', activeTocListener)
+
+    return () => window.removeEventListener('scroll', activeTocListener)
+  }, [contentRef])
+
+  return (
+    <>
+      <NextSeo
+        title={name}
+        titleTemplate={'%s | TinaCMS Docs'}
+        description={excerpt}
+        openGraph={{
+          title: name,
+          description: excerpt,
+          images: [openGraphImage(name, ' | TinaCMS Docs')],
+        }}
+      />
+      <DocsLayout navItems={props.docsNav}>
+        <DocsGrid>
+          <DocGridContent ref={contentRef}>
+            <h1>TinaCMS Packages</h1>
+            <hr />
+            <ul>
+              {props.packages.map( p => {
+                return (
+                  <li><a href={p.link}>{p.name}</a></li>
+                )
+              }
+            )}
+            </ul>
+          </DocGridContent>
+        </DocsGrid>
+      </DocsLayout>
+    </>
+  )
+}
 
 export const getStaticProps: GetStaticProps = async function(props) {
   try {
-    return await getDocProps(props, 'index')
+    const { preview, previewData }: any = props
+
+    const previewProps = await getJsonPreviewProps(
+      'content/toc-doc.json',
+      preview,
+      previewData
+    )
+
+    const docsNavData = previewProps.props.file.data
+
+    const file = await readJsonFile(
+      path.resolve(process.cwd(), './content/packages.json')
+    )
+
+    const packagePages = file.packages.map( p => ({
+      name: p.name,
+      link: p.readme ? `https://tinacms.org/packages/${p.name}` : p.link
```

Comment:
Let's remove the domain from this so that it works in staging or locally
```suggestion
      link: p.readme ? `/packages/${p.name}` : p.link
```